requirements.txt

```diff
@@ -1,3 +1,3 @@
 sphinx>2,<3
-sphinx-hoverxref
+git+git://github.com/readthedocs/sphinx-hoverxref.git@master
```

Comment:
Hmm... not keen on this. `master` seems to be a wip type of branch. So, who knows, it may at some point contain something that could break things for us. You say that they have fixed the issue numref. Is that this commit: https://github.com/readthedocs/sphinx-hoverxref/commit/3558a67e56bf27f08d52425bbcd1d63989c6224e? If so, and if anything, we might want to point to that one rather than `master` itself. However, my preference would on using an official version of sphinx-hoverxref.

Reply:
Yes, that's the commit.  I've tried using just the plain `sphinx-hoverxref` requirement in the file, and it didn't seem to be building it in properly (either that or the cache clearing / readthedocs wiping wasn't being very thorough?).  I'm not sure what the "official" sphinx-hoverxref would point to other than the master branch though?

Reply:
I guess I could ask when the next release will be (seems to be quite frequently) so then we can tie it to a tag - that's probably cleaner?

Reply:
The official sphinx-hoverxref version is `0.3b1` (as confirmed by my local copy of sphinx-hoverxref and https://github.com/readthedocs/sphinx-hoverxref/releases). So, yes, ideally we would ask them for a new release with the numref fix and keep using `sphinx-hoverxref` rather than `git+git://github.com/readthedocs/sphinx-hoverxref.git@master` or something else.

Reply:
They're aiming for another release next week ... fingers crossed :)

Reply:
Great. Ping us again in a week's time. :)